front/pages/api/v1/w/[wId]/data_sources/[name]/documents/[documentId]/index.ts

```diff
@@ -279,6 +289,33 @@ async function handler(
         });
       }
 
+      const whitelistedWorkspaceIds =
+        RUN_DOCUMENT_TRACKER_FOR_WORKSPACE_IDS.split(",");
```

Comment:
cc @spolu, I did env-var-based poor man's feature flagging, does that sound OK ? Allows to merge progressively but keep it no-op for prod